Source/MediaInfo/Audio/File_Aac_GeneralAudio.cpp

```diff
@@ -306,13 +306,13 @@ void File_Aac::program_config_element()
         if (!Infos["Format_Settings_PS"].empty())
         {
             Infos["Format_Profile"]=__T("HE-AACv2");
-            Ztring Channels=Infos["Channel(s)"];
-            Ztring ChannelPositions=Infos["ChannelPositions"];
-            Ztring SamplingRate=Infos["SamplingRate"];
             Infos["Channel(s)"]=__T("2");
             Infos["ChannelPositions"]=__T("Front: L R");
             if (MediaInfoLib::Config.LegacyStreamDisplay_Get())
             {
+                const Ztring Channels = Infos["Channel(s)"];

```

Comment:
`Channels` and `ChannelPositions` are modified in the meanwhile, so they don't behave same.
Fixed in https://github.com/MediaArea/MediaInfoLib/pull/703